### DISCUSSION TITLE
Problema Termostato netatmo

### DISCUSSION THREAD

**Discussion by Capirex666** [indomus/forum#1224] [GENERALE - Discussioni generiche]
Buongiorno a tutti.
Ho un problema con il termostato netatmo e home assistant.
Fino alla versione 2021.11.5 il termostato funziona alla perfezione, ma se aggiorno alla ultima versione il termostato mi da non disponibile.
Sono giorni che provo e giro su internet per vedere di capire il problema ma niente da fare.
I log mi danno questi errori:

Error while setting up netatmo platform for climate
12:04:57 – (ERRORE) Termostato
Platform netatmo not ready yet: None; Retrying in background in 30 seconds
12:04:56 – (AVVERTIMENTO) Sensore
Platform netatmo not ready yet: None; Retrying in background in 30 seconds
12:04:56 – (AVVERTIMENTO) Selezionare
Platform netatmo not ready yet: None; Retrying in background in 30 seconds
12:04:56 – (AVVERTIMENTO) Luce
Error setting up entry Home Assistant Cloud for netatmo
12:04:56 – (ERRORE) helpers/network.py

Sapete come posso risolvere? 
Grazie e buona giornata
Alessio Giovannini

**Comment by Scialla** [comment]
è un problema comune, l'integrazione ha problemi al riavvio di HA

Risolvi rimuovendo l'integrazione, riavviando HA, reinstallando l'integrazione.

Poi al successivo riavvio potresti riavere il problema ma anche no, in ogni caso risolvi allo stesso modo...

Ah, se non lo hai ancora fatto prima, è probabile che ti cambi il nome delle entità; ti consiglio di adeguare i nomi in eventuali automazioni  e non di rinominare le entità, altrimenti dovrai farlo tutte le volte..

Avendo una automazione che gestisce la chiusura delle valvole all'apertura delle finestre (un casino con Netatmo) è un problema a cui sto dietro da un pò...

**Reply by Capirex666** [reply]
Buongiorno Scialla.
Come ho gia scritto ho fatto tantissime prove, anche rimovento e reinstallando l'integrazione ma niente.
Gli errori sono dopo che ho reinstallato l'integrazione.
Non so più che pesci prendere

**Reply by Scialla** [reply]
Rimosso, riavviato HA, reinstallato integrazione? 
Se faccio così funziona subito..

**Reply by Capirex666** [reply]
Si tante e tante volte, non capisco. Ho provato anche a mettere il client_id e il client_secret in configuration.yaml ma niente.
Sono alla disperazione.
Ho rifatto l'installazione (su un pc di prova) di Home Assistant ma niente.

**Reply by Scialla** [reply]
Strano, così dovrebbe andare; comunque ci sono issue su GitHub. Sappi che, in ogni caso, puoi integrarle anche tramite HomeKit, e li non hanno problemi, anche se hanno degli attributi in meno.
Io le uso integrate in entrambi i modi..

**Reply by Capirex666** [reply]
Scusami Scialla, ma homekit non è per apple?

**Reply by Scialla** [reply]
Si certo, ma Netatmo è un dispositivo compatibile con HomeKit quindi, usando l'integrazione HomeKit Controller, puoi integrare tutti i dispositivi compatibili HomeKit (in un colpo solo).
Ovviamente non dovrà essere già stato associato tramite HomeKit altrimenti devi disassociarlo e associarlo ad HA (per poi eventualmente esporlo con l'altra integrazione Homekit (non controller))

HomeKit Controller:
https://www.home-assistant.io/integrations/homekit_controller/

Homekit:
https://www.home-assistant.io/integrations/homekit

**Reply by Capirex666** [reply]
Grande. Per ora ti ringrazio moltissimo e mi metto a studiare homekit.
